test/parallel/test-fs-promises-file-handle-append-file.js

```diff
@@ -39,6 +39,24 @@ async function validateAppendString() {
   await fileHandle.close();
 }
 
-validateAppendBuffer()
-  .then(validateAppendString)
-  .then(common.mustCall());
+async function doAppendAndCancel() {
+  const filePathForHandle = path.resolve(tmpDir, 'dogs-running.txt');
+  const fileHandle = await open(filePathForHandle, 'w+');
+  try {
+    const buffer = Buffer.from('dogs running'.repeat(512 * 1024), 'utf8');
+    const controller = new AbortController();
+    const { signal } = controller;
+    process.nextTick(() => controller.abort());
+    await assert.rejects(fileHandle.appendFile(buffer, { signal }), {
+      name: 'AbortError'
+    });
+  } finally {
+    await fileHandle.close();
+  }
+}
+
+Promise.all([
+  validateAppendBuffer(),
+  validateAppendString(),
+  doAppendAndCancel(),
+]).then(common.mustCall());
```

Comment:
```suggestion
]);
```

This is no longer needed. By default, an unhandled promise rejection makes the process exit with a non-zero exit code.

Reply:
Thanks! I fixed in [27ae1f8](https://github.com/nodejs/node/pull/56972/commits/27ae1f8cd0b83fd83e1071f38ab5c2119451e98e).

Reply:
@y-hsgw sorry, can you add it back? It is still needed to ensure the process does not exit with 0 due to unsettled promises.

Reply:
I've fixed it!